### DISCUSSION TITLE
是否可以支持使用 GPG 为推送到 GitHub 的提交签名？

### DISCUSSION THREAD

**Discussion by Big-Cake-jpg** [Qexo/Qexo#410] [General]
如同题目所言。在使用 Qexo 之前，我的大部分提交都在用 GPG 签名并且开启了 Vigilant mode（警戒模式）。

签名后的提交在 GitHub 上会有一个 Verified 的 badge，同时会显示作者的 GPG Key ID，但是启用了警戒模式后，未签名的提交（包括开始使用 GPG 签名以前所有未签名的）都会被标记为 Unverified。同时，通过 Qexo 进行的提交由于缺失签名也会被标记为未经验证。

所以是否可以考虑加一个可选项目，允许为 Qexo 创建并推送的提交使用 GPG 签名？

**Comment by am-abudu** [comment]
请告诉我Github是否有相关API

**Comment by Big-Cake-jpg** [comment]
> 请告诉我Github是否有相关API

好吧，看起来没办法支持……不过还是非常感谢您的回复。
